pages/1.11/security/ent/tls-ssl/exhibitor/index.md

```diff
@@ -0,0 +1,94 @@
+menuWeight: 500
+excerpt: Securing DC/OS with a TLS enabled Exhibitor ensemble
+enterprise: true
+---
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+By default, the Exhibitor HTTP service is open to any client that can reach port 8181 on a master node. This page describes a method for protecting the Exhibitor service from unauthorized access. Once enabled, HTTP clients must access Exhibitor through Admin Router; thus applying the Admin Router access control policy to the Exhibitor service.
+
+<p class="message--note"><strong>NOTE: </strong>When accessing Exhibitor through Admin Router (https://master_host/exhibitor), authenticated users must have the <i>dcos:adminrouter:ops:exhibitor</i> privilege</p>
+
+# Securing Exhibitor
+
+The strategy for securing Exhibitor is mutual TLS authentication. In order to secure Exhibitor you must first create a unique root CA certificate. This CA certificate is used to sign various end entity certificates for the Admin Router and Exhibitor services. Creating a public key infrastructure that outputs PEM and Java KeyStore formatted artifacts is not a trivial task. To make this processes easier, a simple tool has been created for producing the necessary files.
+
+<p class="message--note"><strong>NOTE: </strong>This guide is only compatible with clusters which use <i>static</i> master discovery, <i>master_http_loadbalancer</i> is not currently supported. (https://docs.mesosphere.com/1.13/installing/production/advanced-configuration/configuration-reference/#master-discovery-required)</p>
+
+## Using the tool
+
+<p class="message--note"><strong>NOTE: </strong>A working Docker installation is required. If Docker is not available see https://github.com/mesosphere/exhibitor-tls-artifacts-gen/blob/master/README.md for information on running the command natively.</p>
+
+Download the script from the GitHub release page and run it:
+
+```sh
+curl -LsO https://github.com/mesosphere/exhibitor-tls-artifacts-gen/releases/download/v0.4.0/exhibitor-tls-artifacts
+chmod +x exhibitor-tls-artifacts
+./exhibitor-tls-artifacts --help
+```
+
+The expected output is shown below:
+
+    Usage: exhibitor-tls-artifacts [OPTIONS] [NODES]...
+
+    Generates Admin Router and Exhibitor TLS artifacts. NODES should consist
+    of a space separated list of master ip addresses. See
+    https://docs.mesosphere.com/1.13/security/ent/tls-ssl/exhibitor/
+
+    Options:
+    -d, --output-directory TEXT  Directory to put artifacts in. This
+                                 output_directory must not exist.
+    --help                       Show this message and exit.
+
+
+
+### Generating the artifacts
+To generate the TLS artifacts, run the tool with the master node ip addresses as positional arguments. Use the ip addresses found in the `master_list` field of the config.yml. If this file is not available, running `/opt/mesosphere/bin/detect_ip` on each master will produce the correct address.
```

Comment:
I think "the config.yml" is not precise enough. Instead, perhaps "The DC/OS configuration file, `config.yml`.".

`s/on each master/on each master node`.